### ISSUE TITLE
Stacktrace for native crash doesnt appear correctly

### ISSUE THREAD

**Issue by pvinis** [getsentry/sentry-react-native#526]
OS:
- [ ]  Windows
- [X]  MacOS
- [ ]  Linux

_Platform:_
- [X]  iOS
- [ ]  Android

Output of `node -v && npm -v && npm ls --prod --depth=0`

```
% node -v && npm -v && npm ls --prod --depth=0
v10.13.0
6.4.1
SentryBadStack@0.0.1 /Users/pvinis/Desktop/SentryBadStack
├── react@16.6.3
├── react-native@0.57.8
└── react-native-sentry@0.40.2

npm ERR! invalid: xcode@1.0.1-dev /Users/pvinis/Desktop/SentryBadStack/node_modules/@sentry/wizard/node_modules/xcode
npm ERR! extraneous: uuid@3.0.1 /Users/pvinis/Desktop/SentryBadStack/node_modules/@sentry/wizard/node_modules/uuid
```

Config:

```
Sentry.config('https://...@sentry.io/...').install()
```

---
I have following issue:
When a native crash happens, the stacktrace is all in one long line, and not formatted correctly.
It's not very useful, as we need to do the extra step of parsing it before we can see where the problem was.

**Steps to reproduce:**
- `yarn install`
- `open ios/SentryBadStack.xcodeproj`
- switch to `SentryBadStack Release` scheme
- `yarn start`
- press play on xcode
- tap `CRASH JS` button
- see everything is nice
- tap `CRASH NATIVE` button
- see that the stacktrace looks weird, no formatting

repro repo: https://github.com/pvinis/SentryBadStack

**Actual result:**
![crash-native](https://user-images.githubusercontent.com/100233/50345494-9bfa8880-052f-11e9-99f4-99c21b51de91.png)

**Expected result:**
Something more like
![this](https://blog.sentry.io/img/post-images/react-native/mixed-stacktraces.png)

**Comment by HazAT** [comment]
Can you try to activate stacktrace merging with 
```
Sentry.config('https://...@sentry.io/...', {
    deactivateStacktraceMerging: false
}).install()
```

We disabled it by default since it could have a performance impact on your app.

**Comment by pvinis** [comment]
I tried, same thing.

**Comment by pvinis** [comment]
@HazAT Actually, I just checked again a bit more carefully. with `deactivateStacktraceMerging: false`, the native errors are not even reported! I was seeing the old ones before and thought they are the same. In my account, I deleted all the issues and tried again. Only js crashes are reported. That's even weirder. Can you reproduce it with my example repo?

**Comment by pvinis** [comment]
# UPDATE: This should be fixed on this [PR](https://github.com/facebook/react-native/pull/23691) and it will land on RN 0.60.0

**Comment by HazAT** [comment]
I am closing all old issues, please if this is still a problem feel free to revive it.
Also, consider moving to our new SDK `@sentry/react-native` if it still happens there please open a new issue.
